### PR TITLE
scripts/network/ping: cleaned up code in baremetal_user_data.py

### PR DESCRIPTION
**Pull request by falcon78921**
## Description

Cleaned up code in ``baremetal_user_data.py``. Updated string formatting
to ``str.format()``. Introduced ``pathlib`` as an alternative to ``os.path``.

Signed-off-by: James McClune <jmcclune@mcclunetechnologies.net>

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

## How Has This Been Tested?

Checked the syntax using PythonBuddy. If there are any errors, please let me know
and I'll fix. :)

### CODE REVIEW

**Comment by falcon78921** [comment]
@DaanHoogland Thanks Daan! :)

**Comment by falcon78921** [comment]
@DaanHoogland I think some of the CI tests need updated to Python3. The ``.travis.yml`` is only testing on 2.7:

https://github.com/apache/cloudstack/blob/master/.travis.yml#L25

I did change some of the Python modules (added ``pathlib``). Is there a place in the CI tests that need updated? I'm seeing ``AttributeErrors`` in some of the Travis tests (Job ``#15120.8``): https://travis-ci.org/apache/cloudstack/jobs/509174933

~~~
======================================================================
ERROR: test suite for <class 'integration.component.test_lb_secondary_ip.TestAssignLBRule'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/.local/lib/python2.7/site-packages/nose/suite.py", line 209, in run
    self.setUp()
  File "/home/travis/.local/lib/python2.7/site-packages/nose/suite.py", line 292, in setUp
    self.setupContext(ancestor)
  File "/home/travis/.local/lib/python2.7/site-packages/nose/suite.py", line 315, in setupContext
    try_run(context, names)
  File "/home/travis/.local/lib/python2.7/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "/home/travis/build/apache/cloudstack/test/integration/component/test_lb_secondary_ip.py", line 108, in setUpClass
    cls.zone.id,
AttributeError: 'str' object has no attribute 'id'
~~~

So far, I don't see anything related to my PR in these errors. If I'm missing something, please let me know.

File: scripts/network/ping/baremetal_user_data.py [hunk 1]
```diff
@@ -15,82 +17,75 @@
 # specific language governing permissions and limitations
 # under the License.
 
-'''
-Created on Jul 2, 2012
+# Created on Jul 2, 2012
+# @author: frank
```
Comment (rhtyd) [inline scripts/network/ping/baremetal_user_data.py]: Remove the created/author comments.

Comment (falcon78921) [reply scripts/network/ping/baremetal_user_data.py]: @rhtyd Those author comments were there previously. They're now gone :)

**Comment by falcon78921** [comment]
@rhtyd If there's anything else you need, please let me know :)

**Comment by rhtyd** [comment]
Confirmed, current systemvmtemplate based on Debian 9 does not have python 3 by default:
Screenshot from 2019-05-27 18-04-42

-1 The change may be breaking, the bigger change would be to replace all of the VR programming codebase (about 2-3k loc) with new rpc mechanism.

**Comment by falcon78921** [comment]
@rhtyd Do you want me to close?

**Comment by falcon78921** [comment]
Made corrections in regards to `pathlib` usage

**Pull request closed without merging**